gufe/protocols/protocolunit.py

```diff
@@ -270,11 +270,10 @@ def execute(self, *,
         ----------
         context : Context
             Execution context for this `ProtocolUnit`; includes e.g. ``shared``
-            and ``scratch`` `Path`s.
+            and ``scratch`` `Path` s.
```

Comment:
Oh there's the one you mentioned. What's puzzling to me is all our use of "`ProtocolUnit`s" in other docstrings on the same object doesn't appear to trigger sphinx's warning at all. So weird. I guess because they aren't defined yet by the time the strings are parsed?